design-toolkit/components/src/components/color-picker/types.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import type { ForwardedRef, ReactNode } from 'react';
+import type {
+  ColorSwatchPickerItemProps as AriaColorSwatchPickerItemProps,
+  ColorSwatchPickerProps as AriaColorSwatchPickerProps,
+  Color,
+} from 'react-aria-components';
+
+export interface ColorPickerState {
+  /** The current color value of the color picker. */
+  color: Color | null;
+  /** Sets the current color value of the color picker. */
+  setColor(color: Color | null): void;
+}
+
+export interface ColorPickerProps extends AriaColorSwatchPickerProps {
+  options: AriaColorSwatchPickerItemProps['color'][];
+  ref?: ForwardedRef<HTMLDivElement>;
+}
+
+export interface ColorSwatchPickerProps
+  extends Omit<ColorPickerProps, 'options'> {
+  children?: ReactNode;
```

Comment:
No need to redefine `children` it's already available on the inherited props. Unless there's a change in functionality(?)

Reply:
Agreed.

Reply:
Apologies, I wasn't clear. I meant I believe that `ColorPickerProps` already has the concept of `children`. I was trying to understand why it needed to be added at all

Reply:
Removed.